tasks/interfaces.yml

```diff
@@ -1,22 +1,17 @@
 ---
-- name: debian network configuration
+- name: Configure Debian networking
   template:
-    backup: no
+    src: interfaces.j2
```

Comment:
Hmm, was meinst du?
Die destination ist doch angegeben und in templates checkt ansible per default.